### DISCUSSION TITLE
No lineage data seen when accessing the UI from a remote client?

### DISCUSSION THREAD

**Discussion by xgboosted** [AbsaOSS/spline-ui#146] [Q&A]
All works fine in the localhost, but it says "Server is unavailable." if I try to access the Spline UI by browsing http://xxx.xxx.xxx.xxx:9090/app/events/list from a remote PC

I have unblocked ports 22 and 80, but still the same.

**Comment by wajda** [comment]
You should unblock port `9090` on `xxx.xxx.xxx.xxx` host, and a respective port on the host where the Spline REST gateway i running.
The Spline UI doesn't route the HTTP requests through its own server side. Instead it makes REST requests to the Spline Consumer API directly from the user browser.

**Reply by xgboosted** [reply]
The Spline REST gateway is running on the same xxx.xxx.xxx.xxx host. Did you mean I should unblock port 9090 on the remote PC from where I am trying to browse http://xxx.xxx.xxx.xxx:9090/app/events/list ?

**Comment by wajda** [comment]
The user PC (a browser) has to have access to both Spline UI and Spline REST host/port. So if both apps are running on the same xxx host they are certainly sitting on different ports. In your case it seems to be 9090 for UI and some other port for REST. Both ports have to be open and accessible to the users.
